hadoop-hdds/docs/content/feature/SCM-HA.md

```diff
@@ -188,7 +188,7 @@ SCM HA uses Apache Ratis to replicate state between the members of the SCM HA qu
 
 This replication process is a simpler version of OM HA replication process as it doesn't use any double buffer (as the overall db thourghput of SCM requests are lower)
 
-Datanodes are sending all the reports (Container reports, Pipeline reports...) to *all* the Datanodes parallel. Only the leader node can assign/create new containers, and only the leader node sends command back to the Datanodes.
+Datanodes are sending all the reports (Container reports, Pipeline reports...) to *all* the Datanodes in parallel. Only the leader node can assign/create new containers, and only the leader node sends commands back to the Datanodes.
```

Comment:
Wei-Chiu, this typo "*all* the Datanodes" is missed.

Reply:
And an extra comment about "### Current SCM HA Security limitation" section. 

 I think we can remove the first item, and change the second item to
"Un-secure HA cluster upgrade to secure HA cluster is not supported."

Reply:
Got it. Looks like the Mandarin doc is more accurate. Interesting.